### ISSUE TITLE
Voice alert 'under attack' when rival creatures are fighting

### ISSUE THREAD

**Issue by Loobinex** [OpenDungeons/OpenDungeons#1141]
In version git20160318 I received voice warnings that I was 'under attack', which alarmed me. However I found that it was just my creatures infighting.
When my own creatures are fighting among themselves I should not hear the voice message 'We're under attack'.

It would be best to replace it with its own voice alert.

**Comment by hwoarangmy** [comment]
Yes, I had also seen that. Thanks for opening the issue

**Comment by hwoarangmy** [comment]
ATM, we have no better suited message for that. When we will have, we can replace it.

**Comment by Loobinex** [comment]
I think any random placeholder sound is better than a misleading one, but fine.

Do you already have a script for the voice actor for this alert? In DK it was "Your creatures are fighting among themselves".

As this is a common notification make sure it is neutral in tone and doesn't contain a joke, because otherwise it will become jarring. Perhaps something like 'A fight has broken out.' or 'Some of your creatures are trying to kill each other.'

**Comment by hwoarangmy** [comment]
@eugeneloza do you have somthing in stock about this ?

**Comment by eugeneloza** [comment]
@hwoarangmy no problem :), but there are two variants:
1. Re-record the whole phrase set. That's better quality, but requires modifying the script to host most recent changes. Like pay-day phrase and etc. see here http://forum.freegamedev.net/viewtopic.php?p=61717#p61717
2. Add just a few phrases. I'm afraid I've lost the specific filters setup when my Adobe Audition experienced a deadly bug :) So it'll sound a little bit 'different' than the current Keeper Groan. But might do as a temporary dummy patch until the voice acting script is in final stage.

**Comment by hwoarangmy** [comment]
Well, that's your call to choose which one is better for you (regarding available time and what you want to do). For my part, I'm ok with a temporary solution if you prefer.

**Comment by eugeneloza** [comment]
For me it's better to re-record everything correctly (plus fix some issues of the previous recording), than doing something flawed :)
But if we're up to re-recording, let's re-check the scenario again. Maybe I should open an Issue?

**Comment by hwoarangmy** [comment]
I guess it would be easier to open an issue and ask what you need. You might also have thought about new stuff to record...

**Comment by eugeneloza** [comment]
Yeah, I have a few new ideas, I'll include them in the latest script and open the issue today.
Let's do it :D
